src/traces/bar/cross_trace_calc.js

```diff
@@ -745,6 +799,18 @@ function collectExtents(calcTraces, pa) {
         return String(Math.round(roundFactor * (p - pMin)));
     };
 
+    // Find min and max size axis extent for each position
+    // This is used for rounded bar corners, to carry rounding
+    // down to lower bars in the case of stacked bars
+    var sMinByPos = {};
+    var sMaxByPos = {};
+
+    // Check whether any trace has rounded corners
+    var anyTraceHasCornerradius = calcTraces.map(function(x) {
+        var trace = x[0].trace;
+        return 'marker' in trace && Boolean(trace.marker.cornerradius);
```

Comment:
Why should we use `Boolean` here? 
Please note 
```js
Boolean(0)
false

Boolean("0")
true
```

Reply:
Also the `.map().some()` could be just a `.some()`

Reply:
Addressed